src/Commands/FailedJobs.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace ActiveCollab\JobQueue\Commands;
+
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Exception;
+
+/**
+ * @package ActiveCollab\JobQueue\Commands
+ */
+class FailedJobs extends Command
+{
+    /**
+     * Configure command
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('failed_jobs')
+             ->setDescription('List failed jobs grouped by type and date');
+    }
+
+    /**
+     * @param  InputInterface  $input
+     * @param  OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $connection = $this->getDatabaseConnection($input);
+
+            $event_types = $connection->executeFirstColumn('SELECT DISTINCT(`type`) FROM `jobs_queue_failed`');
```

Comment:
Extend Queue interface with a method that returns failed job statistics. Key is job type, value is an array where keys are dates and values are number of failed jobs on that particular day.